scipy/fft/_basic.py

```diff
@@ -389,10 +389,123 @@ def irfft(x, n=None, axis=-1, norm=None, overwrite_x=False):
 
 
 def hfft(x, n=None, axis=-1, norm=None, overwrite_x=False):
+    """
+    Compute the FFT of a signal that has Hermitian symmetry, i.e., a real
+    spectrum.
+    Parameters
```

Comment:
It looks like you are still missing a blank line above Parameters here and in `ihfft`

Reply:
Already fixed in b31f7d772bf9ce8414bfcc6c69f2755b84b2a0f9. I can squash the commits if you prefer though.

Reply:
I suppose ideally one might squash the last two DOC commits into one, but I don't think it is a big issue either way.

@larsoner : For my own education, to what extent does SciPy typically encourage rebasing/squashing commits? I think there is a tradeoff between having a concise history of clean commits vs. having excessive rebase requests annoy or discourage contributors.

Reply:
Historically we used to make people squash. Now that maintainers can squash+merge I usually don't bother having contributors do it since I can just do it for them on merge, unless they wants to keep more than a single-commit history or have already taken the time to squash themselves. Sometimes it's also useful for maintaining credit (see https://github.com/scipy/scipy/pull/9876#issuecomment-509448809), in which case rebase+merge is useful.